src/signed-http-client.js

```diff
@@ -3,6 +3,11 @@ const aws4 = require('aws4');
 
 let baseurl = process.env.ApiBaseUrl;
 const region = process.env.AWS_REGION || 'eu-west-1';
+let headers = {};
+
+const setGlobalHeaders = header => {
```

Comment:
You're only pushing 1 header here so I don't think it  makes sense to have a name with plural. Perhaps `pushGlobalHeader` to conform with the logger library `pushGlobalProperty`?

Reply:
Oh the naming is maybe a bit of, but it's possible to "set" multiple headers.

Reply:
RIght, yes of course :) So in essence you're pushing a single header at a  time, but you can call it multiple times? Or are you saying that you can push multiple headers in 1 method call to this library?

Reply:
you can push multiple, example:

```
setGlobalHeaders(
  {
    'header-1': 'value1', 
    'header-2': 'value2 
  }
)
```

Reply:
Right, okay maybe plural is a good way then. I still think add or push is better naming since it's not setting (setting to me, means that you always overwrite the existing value in that variable)